cloud/amazon/ec2_eip.py

```diff
@@ -58,6 +58,12 @@
     required: false
     default: false
     version_added: "2.0"
+  private_ip_address:
```

Comment:
@wimnat I figured so. I am hoping to get this on boto3 at some point so that makes sense.